### DISCUSSION TITLE
Possible webcast with Appeon

### DISCUSSION THREAD

**Discussion by bruce-armstrong** [OpenSourcePFCLibraries/2022#7] [General]
@TABUllrich @mjl38 @patrice-domange @MBalent @kberghall

In a discussion with Armeen about doing a webcast on PFC.  I don't want to be the only one participating.  I'd like to ask anybody that has contributed in the past to also participate.  Please let me know if that would be something you would be interested in doing.

**Comment by Malachosa** [comment]
Absolutely, I’m about to embark on a project to upgrade all our apps to the
latest release of the PFC and PB 2022, and as part of that hope to
contribute any bugs we at BC Ferries have found.

**Reply by bruce-armstrong** [reply]
What timezone are you in?

**Comment by TABUllrich** [comment]
I would prefer to be in the background. My English is not very good. ;-)



Rene Ullrich
Operative Anwendungsentwicklung / Informationstechnologie
T +49 361 7447 111
F +49 361 7447 302

Thüringer Aufbaubank | Anstalt öffentlichen Rechts
Gorkistraße 9 | 99084 Erfurt

aufbaubank.de<http://aufbaubank.de/>
facebook.com/TAB.ThueringensFoerderbank<http://facebook.com/TAB.ThueringensFoerderbank>
twitter.com/aufbaubank <http://twitter.com/aufbaubank>
linkedin.com/company/aufbaubank<http://linkedin.com/company/aufbaubank>
xing.com/companies/thüringeraufbaubank<http://xing.com/companies/thüringeraufbaubank>

Amtsgericht Jena HRA 102 084

Informationen zum Umgang mit Ihren Daten in der Thüringer Aufbaubank und zu Ihren Rechten nach der EU-Datenschutz-Grundverordnung finden Sie im Internet auf der Seite www.aufbaubank.de/Impressum-und-Datenschutz<http://www.aufbaubank.de/Impressum-und-Datenschutz>
Von: Bruce Armstrong ***@***.***>
Gesendet: Mittwoch, 24. Januar 2024 04:40
An: OpenSourcePFCLibraries/2022 ***@***.***>
Cc: Ullrich, Rene ***@***.***>; Mention ***@***.***>
Betreff: EXTERN: [OpenSourcePFCLibraries] Possible webcast with Appeon (Discussion #7)


@TABUllrich<https://github.com/TABUllrich> @mjl38<https://github.com/mjl38> @patrice-domange<https://github.com/patrice-domange> @MBalent<https://github.com/MBalent> @kberghall<https://github.com/kberghall>

In a discussion with Armeen about doing a webcast on PFC. I don't want to be the only one participating. I'd like to ask anybody that has contributed in the past to also participate. Please let me know if that would be something you would be interested in doing.

**Comment by MBalent** [comment]
I can participate.  Use the PFC every day in my current role.

Matt Balent

**Reply by bruce-armstrong** [reply]
What timezone are you in?

**Comment by kberghall** [comment]
I can participate, we use PFC extensively. We are in Central time zone.

**Comment by MBalent** [comment]
Eastern

**Comment by bruce-armstrong** [comment]
Ok, so I've been tied up and haven't been able to follow up on this.  The idea is that I do a presentation on what PFC is, why people would want to use it and how people can contribute to it.  I would ask each of you to record a short (a few minutes) video indicating a) why you participate in the PFC project and b) a sample of the contributions you've made.  Please let me know if you're still interested in doing that.

My schedule eases up after Easter and I can get a chance to work on the presentation.

**Comment by MBalent** [comment]
Sounds good, I'll try to get something together in a couple of weeks.
First week of April is busy for me

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by bruce-armstrong** [comment]
Could you guys go into the settings under "People" and set your participation in the project as "Public" rather than "Private".  One of the things I want to mention as an advantage of using PFC is that there are a number of people supporting the project.  If people look at the project page right now though the only person they see is me.

![image](https://github.com/OpenSourcePFCLibraries/2022/assets/6356830/4d1297bd-c3e1-45a6-9586-10ba78689959)

**Comment by bruce-armstrong** [comment]
OK, so I got distracted for a bit.  This is actually happening though.  At this point I've submitted it as a presentation for Elevate 2024.  I've completed the PPT presentation.  If you'd still like to participate, please contact me directly at bruce_at_brucearmstrong_dot_org.

Sorry for the delay.
